Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/turret.yml

```diff
@@ -0,0 +1,293 @@
+- type: entity
+  parent: BaseWeaponTurret
+  id: WeaponDisablerTurret
+  name: crowd control turret
+  suffix: NanoTrasen
+  description: A disabler based projectile auto-turret, brought to you by S.E.S.W.C.
+  components:
+    - type: BallisticAmmoProvider
+      proto: BulletDisabler
+      capacity: 500
```

Comment:
I'm not a fan how energy based turret is handled here. I'd argue that you should take a look at battery weapons:
```
    - type: Battery
      maxCharge: 10000
      startingCharge: 10000
    - type: ProjectileBatteryAmmoProvider
      proto: BulletDisabler
      fireCost: 100

```
Further more I'm not sure what is the point of "non-lethal" turrets: it absolutely will continue firing at it's target until the target is dead.

Reply:
The fire rate of the turret prevents them from outright killing them

Reply:
this is what i am getting from that  

![image](https://github.com/new-frontiers-14/frontier-station-14/assets/150621839/cb67a6fa-cbb2-4ae7-a907-78237393c752)

Reply:
seem to be shooting BOTH bullets and lasers, I notice that when a laser would hit me but I see a bullet pass me

Reply:
![image](https://github.com/new-frontiers-14/frontier-station-14/assets/150621839/5510b6a8-27c5-4535-83e8-ec2d72a30ecf)

Reply:
> seem to be shooting BOTH bullets and lasers, I notice that when a laser would hit me but I see a bullet pass me

This is because you need new parent for your turret. One that doesn't have this stuff backed in it:
```
    - type: BallisticAmmoProvider
      proto: CartridgeCaselessRifle
      capacity: 500
 ```

Reply:
I will look into it after the funeral.  I guess in a way this could make duel weapon turrets now like this which is interesting within it self